Mathlib/Data/Vector/Defs.lean

```diff
@@ -130,6 +187,14 @@ def map₂ (f : α → β → φ) : Vector α n → Vector β n → Vector φ n
 def replicate (n : ℕ) (a : α) : Vector α n :=
   ⟨List.replicate n a, List.length_replicate⟩
 
+@[simp] theorem toVector_replicate :
+    ∀ n (a : α), (replicate n a).toVector = _root_.Vector.replicate n a :=
+  fun _ _ => rfl
```

Comment:
```suggestion
@[simp] theorem toVector_replicate (n : ℕ) (a : α) :
    (replicate n a).toVector = _root_.Vector.replicate n a :=
  rfl
```

Reply:
Not always able to move this into before the `:`, but can do when I don't need to destructure it.